javascript/packages/orchestrator/src/chainSpec.ts

```diff
@@ -74,9 +74,13 @@ export function clearAuthorities(specPath: string) {
     if (runtimeConfig?.collatorSelection)
       runtimeConfig.collatorSelection.invulnerables = [];
 
-    // Clear staking
+    // clear staking
     if (runtimeConfig?.staking) {
-      stakingBond = BigInt(runtimeConfig.staking.stakers[0][2]);
```

Comment:
if the staking genesis is set but with an empty vec of stakers (which is totally OK), accessing `stakers[0][2]` would panic.